docs/src/main/asciidoc/config.adoc

```diff
@@ -137,3 +134,8 @@ $ gcloud beta runtime-config configs variables set \
 ....
 $ curl -XPOST http://myapp.host.com/actuator/refresh
 ....
+
+=== Sample
+
+A https://github.com/spring-cloud/spring-cloud-gcp/tree/master/spring-cloud-gcp-samples/spring-cloud-gcp-config-sample[sample application]
+and a https://codelabs.developers.google.com/codelabs/cloud-spring-runtime-config/index.html[codelab] are available.
```

Comment:
Must be on a single line: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line